lib/resources/example_json/r4_examples_care_plan.rb

```diff
@@ -46,6 +46,68 @@ module Resources
       }
     }.freeze
 
+    R4_CARE_PLAN_PATIENT_ENTRY ||= {
+      'resourceType': 'CarePlan',
+      'id': '197286315',
+      'meta': {
+        'versionId': '1',
+        'lastUpdated': '2020-04-05T04:00:46.000Z'
+      },
+      'text': {
+        'status': 'additional',
+        'div': '&lt;div xmlns="http://www.w3.org/1999/xhtml"&gt;&lt;p&gt;&lt;b&gt;'\
+               'Care Plan&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;Patient&lt;/b&gt;'\
+               ': SMART, NANCY&lt;/p&gt;&lt;p&gt;&lt;b&gt;Title&lt;/b&gt;: Pregnancy Summary Document'\
+               '&lt;/p&gt;&lt;p&gt;&lt;b&gt;Description&lt;/b&gt;: SMART, NANCY&#10;DOB: 08/11/1980&#9;'\
+               'Age: 39 Years&#9;MRN: 00000006930&#10;---------------------------------------------------'\
+               '&#10;&#9;&#9;&#9;&#9;SMART, NANCY DOB: 08/11/80&#10;Pregnancy Summary (Date of Report: 04/04/20)'\
+               '&#10;&#10;G 1 P 0 (0,0,0,0)&#10;Gestation: --&#9;LMP (from pt. history): --&#10;EDD: --&#9;EDD/EGA'\
+               'Method: Date of Conception&#9;EGA: --&#10;&#10;-----------------------------------------------'\
+               '&#10;&#9;&#9;&#9;&#9;SMART, NANCY DOB: 08/11/80&#10;Antepartum Note&#10;&#10;&#9;'\
+               'No antepartum notes have been documented&#10;&#10;--------------------------------'\
+               '-&#10;&#9;&#9;&#9;&#9;SMART, NANCY DOB: 08/11/80&#10;Problems (Active Problems Only)&#10;'\
+               'Gestational diabetes mellitus, class A&gt;1&lt; (Related to Pregnant problem) (SNOMED CT: 124602011,'\
+               'Onset: 03/05/20)&#10;&#9;Comment: Problem added by Discern Expert (SYSTEM, SYSTEM Cerner on 03/05/20)'\
+               '&#10;&#10;Gestational hypertension (Related to Pregnant problem)'\
+               '(SNOMED CT: 495022012, Onset: 03/05/20)&#10;&#9;Comment:'\
+               'Problem added by Discern Expert (SYSTEM, SYSTEM Cerner on 03/05/20)&#10;&#10;'\
+               'Pregnant (SNOMED CT: 191073013, Onset: 06/12/19)&#10;&#10;--------------------------------------------'\
+               '&#10;&#9;&#9;&#9;&#9;SMART, NANCY DOB: 08/11/80&#10;Risk Factors and Genetic Screening&#10;'\
+               'Risk Factors (Current Pregnancy)&#10;Risk Factors: Diabetes, gestational,'\
+               'type A1, Gestational hypertension&#10;Risk Factors Comments: --&#10;&#10;Ethnic Screening&#10;'\
+               '&lt;/p&gt;&lt;p&gt;&lt;b&gt;Status&lt;/b&gt;: Active&lt;/p&gt;&lt;p&gt;&lt;b&gt;Intent&lt;/b&gt;'\
+               ': Plan&lt;/p&gt;&lt;p&gt;&lt;b&gt;Category&lt;/b&gt;: Assessment and Plan of Treatment&lt;/p&gt&'\
+               'lt;p&gt;&lt;b&gt;Author&lt;/b&gt;: SYSTEM, SYSTEM Cerner&lt;/p&gt;'\
```

Comment:
For div, instead of &lt / &gt tags use the content within single quote

Eg:

'div': '<div xmlns="http://www.w3.org/1999/xhtml"><p><b>Schedule</b></p><p><b>Status</b>: Active</p>'\
                 '<p><b>Service Type</b>: Established Patient</p><p><b>Practitioner</b>'\
                 ': Cerner Test, Physician - Primary Care Cerner</p></div>'

Reply:
Fixed